### PR TITLE
Remove extra includes from OV constant.hpp

### PR DESCRIPTION
**Pull request by vurusovs**
### Details:
 - Additional changes after https://github.com/openvinotoolkit/openvino/pull/22182 (better review only 9f00165f113448af2d453c3eda7ae5a85c84afed)

### CODE REVIEW

File: src/core/src/runtime/aligned_buffer.cpp [hunk 1]
```diff
@@ -2,70 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/runtime/aligned_buffer.hpp"
-
 #include <algorithm>
 #include <memory>
 
-#include "ngraph/util.hpp"
+// #include "ngraph/util.hpp"
```
Comment (praasz) [inline src/core/src/runtime/aligned_buffer.cpp:8]: Remove it?

File: src/core/src/runtime/aligned_buffer.cpp [hunk 2]
```diff
@@ -2,70 +2,12 @@
...
-}
-}  // namespace ngraph
-
-namespace ov {
-AttributeAdapter<std::shared_ptr<ngraph::runtime::AlignedBuffer>>::AttributeAdapter(
-    std::shared_ptr<ngraph::runtime::AlignedBuffer>& value)
-    : DirectValueAccessor<std::shared_ptr<ngraph::runtime::AlignedBuffer>>(value) {}
-}  // namespace ov
-NGRAPH_SUPPRESS_DEPRECATED_END
+// #include "openvino/util/log.hpp"
```
Comment (praasz) [inline src/core/src/runtime/aligned_buffer.cpp:10]: Remove?

File: src/common/snippets/src/lowered/linear_ir.cpp [hunk 3]
```diff
@@ -2,6 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "ngraph/graph_util.hpp"
```
Comment (praasz) [inline src/common/snippets/src/lowered/linear_ir.cpp:5]: Or add openvino include?

File: src/tests/ov_helpers/ov_models/src/utils/ov_helpers.cpp [hunk 4]
```diff
@@ -17,6 +17,7 @@
 #include "openvino/op/util/attr_types.hpp"
 #include "openvino/pass/constant_folding.hpp"
 #include "openvino/runtime/tensor.hpp"
+// #include "ngraph/util.hpp"
```
Comment (ilya-lavrenov) [inline src/tests/ov_helpers/ov_models/src/utils/ov_helpers.cpp:20]: drop?

**Comment by ilya-lavrenov** [comment]
can we finish with this PR?
It looks important to decouple old and new public headers

**Comment by ilya-lavrenov** [comment]
does it still contain useful changes or can be closed?

**Comment by vurusovs** [comment]
For the moment PR doesn't contain valuable changes, so closing it

**Pull request closed without merging**